Assets/HoloToolkit/ARCA/Scripts/Recording/ReplayKitRecorder.cs

```diff
@@ -0,0 +1,152 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.using UnityEngine;
+
+using System;
+using UnityEngine;
+using HoloToolkit.Unity.InputModule;
+#if PLATFORM_IOS
+using UnityEngine.iOS;
+using UnityEngine.Apple.ReplayKit;
+#endif
+using UnityEngine.UI;
+
+namespace HoloToolkit.ARCapture
+{
+    public class ReplayKitRecorder : MonoBehaviour
```

Comment:
please add summary blocks to describe the class and the members (not needed for Unity methods like Start / Update)